### ISSUE TITLE
PdoBackend: Error when running cache:setupall after upgrading neos/cache from 5.3.3 to 5.3.4

### ISSUE THREAD

**Issue by drlimbo** [neos/flow-development-collection#1763]
### Description
Error when running `cache:setupall` after upgrading neos/cache from 5.3.3 to 5.3.4

`Neos_Media_ImageSize:
Creating database tables "cache" & "tags"...
ERROR: Could not create cache tables with DSN "sqlite:/..../Data/Temporary/Production/Cache/Data/Neos_Media_ImageSize.sqlite". 
PDO error: SQLSTATE[HY000]: General error: 1 table "cache" already exists`

### Steps to Reproduce

1. Setup PDOBackend Cache using an SQLite database
2. Clear all Caches
3. Run `cache:setupall`
4. 💥

#### Expected behavior
Cache should be setup. 

#### Actual behavior
Throws an Error, but table is created. 

### Affected Versions
Neos: 5.2.8, 5.3.4

already discussed with @bwaidelich [in this Slack thread](https://neos-project.slack.com/archives/C050C8FEK/p1568107878288200)

**Comment by bwaidelich** [comment]
@drlimbo Thanks for creating the ticket. I reproduced this locally with the following `Caches.yaml` configuration:
```yaml
Flow_Mvc_Routing_Route:
  backend: Neos\Cache\Backend\PdoBackend
  backendOptions:
    dataSourceName: 'sqlite:Data/cache.db'
```

Result of `./flow cache:setupall`:

```
...
Flow_I18n_Cldr_Reader_CurrencyReaderCache:
OK
Flow_Monitor:
OK
Flow_Mvc_Routing_Route:
Creating database tables "cache" & "tags"...
ERROR: Could not create cache tables with DSN "sqlite:Data/cache.db". PDO error: SQLSTATE[HY000]: General error: 1 table "cache" already exists
...
```

The reason for this is that the SQLite database is [automatically created upon connection](https://github.com/neos/flow-development-collection/blob/master/Neos.Cache/Classes/Backend/PdoBackend.php#L368) so there is no need to set them up explicitly.

So technically the error is correct – it tries to create tables that exist already.

But I agree that the error message might be confusing and I'll take care of a PR

**Comment by bwaidelich** [comment]
Fixed with #1766
